### ISSUE TITLE
Release 0.7.0.7 to PyPi job failed

### ISSUE THREAD

**Issue by warenick** [BerkeleyAutomation/python-fcl#82]
<img width="884" alt="image" src="https://github.com/user-attachments/assets/7c475ef2-a0fa-4194-8d93-356afdb035fe">
<img width="849" alt="image" src="https://github.com/user-attachments/assets/a550d3ac-0565-4c26-ac97-278cba342690">

**Comment by naxa1ka** [comment]
Up

**Comment by mikedh** [comment]
Yeah looks like the token expired. @mmatl would you mind updating the token in this repository's secrets or adding me as a project admin?

**Comment by mikedh** [comment]
Heh I have a todo in one of my Dockerfiles with a temporary workaround:
```
wget https://github.com/BerkeleyAutomation/python-fcl/releases/download/v0.7.0.7/python_fcl-0.7.0.7-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl && pip install *.whl
```

**Comment by patrickclerc** [comment]
Hello,

Do you have any news concerning the release of the version 0.7.0.7 ? 
We have hosted the builder wheel on our server to test the python-fcl with numpy 2 and it works as expected. 
We would like to install it from the official pypi release instead of our own hosted server. 

Thanks.

**Comment by mmatl** [comment]
@patrickclerc @mikedh sorry for the super-delayed reply. I haven't been a part of the BerkeleyAutomation organization for a while, so I do not have admin access to this repo. You'll need to contact one of the maintainers directly.

**Comment by aniritafc** [comment]
Hi,

Do you have any updates on when version 0.7.0.7 will be available for installation through PyPi?

Thanks!
